enterprise_subsidy/apps/content_metadata/api.py

```diff
@@ -2,6 +2,75 @@
 Python API for gathering content metadata for content identifiers
 during subsidy redemption and fulfillment.
 """
+import logging
+from decimal import Decimal
+
+from enterprise_subsidy.apps.subsidy.constants import CENTS_PER_DOLLAR
+
+from .constants import EDX_PRODUCT_SOURCE, EDX_VERIFIED_COURSE_MODE, EXECUTIVE_EDUCATION_MODE, TWOU_PRODUCT_SOURCE
+
+logger = logging.getLogger(__name__)
+
+
+CONTENT_MODES_BY_PRODUCT_SOURCE = {
+    EDX_PRODUCT_SOURCE: EDX_VERIFIED_COURSE_MODE,
+    TWOU_PRODUCT_SOURCE: EXECUTIVE_EDUCATION_MODE,
+}
+
+
+def price_for_content(content_data):
+    """
+    Helper to return the "official" price for content.
+    The endpoint at ``self.content_metadata_url`` will always return price fields
+    as USD (dollars), possibly as a string or a float.  This method converts
+    those values to USD cents as an integer.
+    """
+    content_price = None
+    if content_data.get('first_enrollable_paid_seat_price'):
```

Comment:
this isn't covered in the current tests - is this really an expected form of price? if so we may wanna add a test